src/components/OperationalStatus/OperationalStatus.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FunctionComponent } from "react";
+import {
+  Text,
+  Box,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableContainer,
+  Tag,
+  TagLabel,
+  Alert,
+  AlertIcon,
+  AlertTitle,
+} from "@chakra-ui/react";
+
+interface Service {
+  name: string;
+  status: string;
+  detected?: string;
+  error?: string;
+}
+
+interface Data extends Service {
+  last_updated?: string;
+  services: Service[];
+}
+
+export interface OperationalStatusData extends Data {
```

Comment:
The convention we use for other components is to call this OperationalStatusProps, that way we know its the props for the OperationalStatus component

Reply:
Updated